telegram/error.py

```diff
@@ -51,6 +51,9 @@ def __init__(self, message):
     def __str__(self):
         return '%s' % (self.message)
 
+    def __reduce__(self):
+        return type(self), (self.message,)
```

Comment:
You can use `self.__class__` instead. Don't know if there are any style conventions on that, but that's what we currently do for `Bot` and it doesn't need a function call ... Same for the others